### ISSUE TITLE
"Fatal error" RStudio crashes systematically

### ISSUE THREAD

**Issue by liliay** [rstudio/rstudio#12103]
Hi there, 

My R session has been terminated suddenly for no reason, and most surprising it is no longer responding and displays a fatal error. 
Here is the problem report log file. 
It seems to be an issue with javascript. Anyone who has faced this issue and solved it to help me figure that out ? Thanks in advance for your insight. 

[rsession-liliay.log](https://github.com/rstudio/rstudio/files/9715465/rsession-liliay.log)

Lilia

**Comment by jonvanausdeln** [comment]
@liliay thank you for reporting your issue!  Is this an issue you see regularly or is this a one time issue?

Rstudio Version and R version information would be helpful

**Comment by liliay** [comment]
Thank you for willing to help. 
I changed computer in february 22, since then I've had this issue 3 times, I just reopenned a new session and worked fine after. 
This time, nothing is working. 
Since my post I reinstalled Rstudio, and still have the same problem (it worked fine for a day and today while running analysis it just crashed again, with the exact same error message).

R version 4.2.1 (2022-06-23)
RStudio 2022.07.2+576 (Not sure what this means)
